dom/data/core.js

```diff
@@ -0,0 +1,64 @@
+var isEmptyObject = require("../../js/is-empty-object/is-empty-object");
+
+var data = {};
+var expando = "can" + new Date();
+var uuid = 0;
+
+var setData = function(name, value) {
+	var id = this[expando] || (this[expando] = ++uuid),
+		store = data[id] || (data[id] = {});
+	if (name !== undefined) {
+		store[name] = value;
+	}
+	return store;
+};
+
+// delete this node's `data`
+// returns the number of keys that were deleted
+var deleteNode = function() {
+	var id = this[expando];
+	var count = 0;
+	if(id) {
+		count = data[id] ? Object.keys(data[id]).length : 0;
```

Comment:
This is checking the opposite... so if you call `domData.set` twice for the same element it will be counted correctly. 

Are you suggesting I use `isEmptyObject` to know when to call `offAfterRemovedNodes` instead of counting at all?